rpcs3/Emu/system_config.h

```diff
@@ -130,7 +129,7 @@ struct cfg_root : cfg::node
 		cfg::_enum<frame_limit_type> frame_limit{ this, "Frame limit", frame_limit_type::_auto, true };
 		cfg::_float<0, 1000> second_frame_limit{ this, "Second Frame Limit", 0, true }; // 0 disables its effect
 		cfg::_enum<msaa_level> antialiasing_level{ this, "MSAA", msaa_level::_auto };
-		cfg::_enum<shader_mode> shadermode{ this, "Shader Mode", shader_mode::async_recompiler };
+		cfg::_enum<shader_mode> shadermode{ this, "Shader Mode", shader_mode::async_with_interpreter };
```

Comment:
Good call, though I need to push a change to precompile the interpreter after the loading shaders view is shown. This will avoid the long first-time hitch when the ubershader is seen by the driver the first time. This is especially noticeable on AMD windows driver.
It's not too difficult, I'll push it out in a day or two.